skfda/preprocessing/dim_reduction/projection/_fpca.py

```diff
@@ -397,3 +397,39 @@ def fit_transform(self, X, y=None, **fit_params):
         """
         self.fit(X, y)
         return self.transform(X, y)
+
+    def get_component_perturbations(self, X, index=0, multiple=30):
```

Comment:
Why `30` as the default value?

Reply:
This is just an arbitrary value, I was thinking about making this attribute not optional. This parameter depends on each dataset and cannot be predetermined.